src/main/scala/com/typesafe/sbt/packager/universal/Archives.scala

```diff
@@ -58,9 +60,10 @@ object Archives {
    *  @param name of output (without extension)
    *  @param mappings included in the output
    *  @param top level directory : NOT USED
+   *  @param options  NOT USED
    *  @return dmg file
    */
-  def makeDmg(target: File, name: String, mappings: Seq[(File, String)], top: Option[String]): File = {
+  def makeDmg(target: File, name: String, mappings: Seq[(File, String)], top: Option[String], options: Seq[String]): File = {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method is longer than 50 lines](https://www.codacy.com/app/nepomukseiler/sbt-native-packager/file/946190549/issues/source?bid=2040925&fileBranchId=2849134#l66)